PocketKit/Sources/PocketKit/Home/TopicChipCell.swift

```diff
@@ -71,6 +88,7 @@ extension TopicChipCell {
 
     static func width(chip: TopicChipPresenter) -> CGFloat {
         let size = chip.attributedTitle?.sizeFitting() ?? .zero
-        return size.width.rounded(.up) + 24
+        let imageSize = chip.iconImage?.size.width ?? .zero
```

Comment:
Use `24` points of padding if imageSize is `0`

Reply:
Fixed as requested.